app/api/v1/resources_on_behalf.rb

```diff
@@ -0,0 +1,48 @@
+require 'services/publish_on_behalf_interactor'
+
+module API
+  module V1
+    # Default options for all API endpoints and versions
+    module ResourcesOnBehalf
```

Comment:
this is unimportant, but: I still don't like the name 'resource on behalf', but I don't want to delay you because of naming. 
This feature is simply the 'publish' api, we came with the "encode an envelope on behalf of the user" as a short explanation for the feature, but it's publishing. 
https://github.com/CredentialEngine/CredentialRegistry/issues/139

Anyway, this is just class naming, no big deal.